src/sadatom/main.cpp

```diff
@@ -257,13 +258,18 @@ int main(int argc, char **argv) {
   int iconf(parser.get<int>("iconf"));
   double conf_R(parser.get<double>("conf_R"));
   int conf_N(parser.get<int>("conf_N"));
+  double conf_barrier(parser.get<double>("conf_barrier"));
   double shift_conf(parser.get<double>("shift_conf"));
 
+  // Truncate radial grid in case of Junquera et al. confinement potential
+  if(iconf==4)
+    Rmax=conf_R;
```

Comment:
```suggestion
  // Truncate radial grid in case of Junquera et al. confinement potential
  if(iconf==4) {
    if(Rmax != conf_R) {
      printf("Using the Junquera et al potential: setting Rmax to conf_R!\n");
      Rmax=conf_R;
    }
  }
```